### DISCUSSION TITLE
100x100 mm through hole version

### DISCUSSION THREAD

**Discussion by mengstr** [markadev/AppleII-VGA#47] [Show and tell]
Thank you for this project.  My 15" Philips Matchline TV (with Antenna, Composite, S-Video, SCART and VGA) didn't sync up nicely with the Apple composite output but had jitter and was jumping all over the place.  So I remade your original PCB into a size that could be made at JLC for $2, populated it,  and of course it works perfectly. ^__^

The colors are a bit off, but that is to be expected considering I used 470, 1K and 2.2K in the DAC. Haven't tried the 80-col mode yet, but I never had that on my original Apple back in the day, so it will just be a nice addition if it works.  Now I just have to re-learn using the Apple again - toy tend to forget a lot in 45 years :-)

![IMG_20240824_165907](https://github.com/user-attachments/assets/7384e7fd-939e-4627-af5c-7dbaec70d669)
![IMG_20240824_234411](https://github.com/user-attachments/assets/9a2aa1ce-5651-474c-92e1-999b216befec)

**Comment by markadev** [comment]
Nice job getting the board size down, and thanks for sharing!

**Reply by vitasam** [reply]
Nice job! I started my own design of 100x100 board for [MPF-II_VGA_card](https://github.com/vitasam/mpf-ii_retro_6502/tree/main/VGA-card) - I want to fit also the VGA connector to the same board

**Comment by emilen72** [comment]
I'm very interested in 100x100 version, any chance to sharing the gerber files?

**Reply by mengstr** [reply]
Not sure if you get my reply when I just add it into the thread here and not specifically to your text. If not you'll have to look further down for it.

**Comment by mengstr** [comment]
Oh, I see that I only got an empty repo for it. I'll push the files later today.  But I have 4 spare boards that I've got no use for so I'd be happy to send you one for free, if you want one drop me an email with your address at 
MATS.ENGSTROM () PROTONMAIL () CH 
Of course replace the () with the relevant characters ;-)

**Reply by emilen72** [reply]
mail sended, tnx

**Comment by mengstr** [comment]
Repo with KiCad files and also Gerbers, made for JLCPCB but probably works for most fabs, available at https://github.com/mengstr/AppleII-VGA100

**Reply by emilen72** [reply]
thanks

**Comment by emilen72** [comment]
﻿Hi Mats,Pcb is arrived to me, no trouble at all :-)Mounted on the fly and is working…I don’t have the LVC245 and use the HTC245 and it works. Pico are unofficially 5,25V tolerant :-)Some pics and many many thanksEmilianoELIl giorno 16 set 2024, alle ore 19:03, Mats Engstrom ***@***.***> ha scritto:﻿
Repo with KiCad files and also Gerbers, made for JLCPCB but probably works for most fabs, available at https://github.com/mengstr/AppleII-VGA100

**Reply by mengstr** [reply]
Nice!, you're very welcome.  I'm glad you got it working - and that someone got use for my extra PCBs instead of it just being stuck in a box of old boards here. :-)
